packages/web-components/@groupby/elements-base/src/decorators.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A property decorator that will set the data initialization state in a class.
+ * It will extend the property's setter to set the initialization state to true.
+ * This expects that the property already has defined accessors and a default value.
+ *
+ * @param initProperty The name of the initialization property.
+ */
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
This is fine for now. Later on, if we have more decorators, we should consider adding a file-wide disable directive at the top:

```typescript
/* eslint-disable import/prefer-default-export */
```